routers/web/repo/projects.go

```diff
@@ -666,3 +681,70 @@ func MoveIssues(ctx *context.Context) {
 
 	ctx.JSONOK()
 }
+
+// MoveColumns moves or keeps columns in a project and sorts them inside that project
+func MoveColumns(ctx *context.Context) {
```

Comment:
I just saw the same function above? Why duplicate

Reply:
d300bfd